datalad_registry/conf.py

```diff
@@ -53,7 +53,7 @@ class BaseConfig(OperationConfig):
         "bids_dataset",
         # === DANDI related extractors ===
         "dandi",
-        # "dandi:files",  # Let's not activate this yet by default
+        "dandi:files",
```

Comment:
for production we would not want it in this shape.

So we will just need to comment out here.